service/auth/deploy/manifests/auth.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright © 2022 sealos.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sealos
+  annotations:
+  labels:
+    name: sealos
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: service-auth
+  namespace: sealos
+spec:
+  ports:
+    - port: 8080
+  selector:
+    app: service-auth
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: service-auth
+  namespace: sealos
+spec:
+  selector:
+    matchLabels:
+      app: service-auth
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: service-auth
+    spec:
+      containers:
+        - name: service-auth
+          image: service-auth:latest
```

Comment:
So we will put this image into cluster image?

Reply:
Cluster image seems to only be responsible for storeing claims of cluster, like a `Deployment` yaml. But the individual image of each container is managed by the registry.

Reply:
yes，what we need a two step ci，one for build service－auth，then the later one build cluster image.